react/features/aeternity/components/TipButton.js

```diff
@@ -287,11 +287,9 @@ class TipButton extends Component<Props, State> {
         const { t } = this.props;
         const amount = aeternity.util.aeToAtoms(this.state.value);
         const url = `${URLS.SUPER}/user-profile/${this.props.account}`;
+        const message = this.state.message;
 
         try {
-            const DEFAULT_MESSAGE = `Appreciation from conference : ${APP.conference.roomName} on ${window.location.host}.`;
```

Comment:
@mike-gusev-ae we can't remove default message until we use contract v1 
ref: https://github.com/aeternity/jitsi-meet/pull/206#pullrequestreview-476456214